os/fs/driver/mtd/smart.c

```diff
@@ -4978,7 +4979,8 @@ static uint32_t smart_journal_get_writeaddress(FAR struct smart_struct_s *dev)
  *
  * Description:
  *   Increase journal sequence.
- *   If current erase block is filled with journal entries, erase it.
+ *   If current erase block is filled with journal entries, we move to the
+ *   next block automatically and erase the current.
```

Comment:
how about the  "erase the current erase block" at the end of the comment ?

Reply:
how about "we automatically move to the next erase block and erase it." ??

Reply:
Using "we automatically move to the next block and erase the current".
Thanks for pointing it out.